algosdk/logic.py

```diff
@@ -0,0 +1,122 @@
+import json
+import os
+
+from . import error
+
+spec = None
+opcodes = None
+
+MAX_COST = 20000
+MAX_LENGTH = 1000
```

Comment:
Should MAX_COST and MAX_LENGTH go in constants.py? Also, is not setting spec and opcodes to avoid processing langspec.json if not needed?

Reply:
done